randovania/games/planets_zebeth/exporter/patch_data_factory.py

```diff
@@ -43,14 +43,24 @@ def _create_pickups_dict(self, pickup_list: list[ExportedPickupDetails], item_in
                 else 0
             )
 
+            pickup_type = "Nothing"
+            if not pickup.other_player:
+                if pickup.original_pickup.name.startswith("Tourian Key"):
+                    pickup_type = "Tourian Key"
+                else:
+                    pickup_type = pickup.original_pickup.name
+
+            acquired_msg = f"{pickup.name} acquired"
+            if not self.players_config.is_multiworld:
+                acquired_msg = item_info[pickup.name]["text_header"].replace(pickup_type, pickup.original_pickup.name)
```

Comment:
what is this supposed to do?

Reply:
It fixes the name of the pickup when it's different than generic pickup
ex : Tourian Key 6 != Tourian Key

Reply:
I'm still not quite following. So your goal is to specifically make this say  `Tourian Key 6 acquired` instead of `Tourian Key acquired`, right?
Cant you change your text data to have `{item}` and then use `.format(item=pickup.original_pickup.name)`?

Reply:
if there's a way to actually clone a dict I wouldn't mind since changing one of the item_data['Tourian Key `x`'] change each of them with the current way it's done

Reply:
why would .format not work for you?

Reply:
because currently it uses item_data to get the acquired text unless I stop using item_data for the header which makes item_data.json only useful for description